packages/api/src/controllers/alerts.ts

```diff
@@ -122,6 +123,53 @@ export const getAlertById = async (
   });
 };
 
+export const getDashboardAlerts = async (
+  dashboardIds: string[] | null,
+  teamId: ObjectId,
+) => {
```

Comment:
style: imo, its not very readable when the 1st arg is null. I'd create two methods `getTeamDashboardAlertsByTile(teamId: ObjectId)` and `getDashboardAlertsByTile(teamId: ObjectId, dashboardId: string)`. we don't need to support the array case.

Reply:
makes sense, updated